spec/cap_decorator_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../cap_decorator'
+
+describe CapitalizeDecorator do
+  before :each do
+    @classroom = Classroom.new('History')
+    @student = Student.new(55, @classroom, 'alexis')
+  end
+  context 'test capitalizer' do
+    it 'should return a name capitalized' do
+      @capitalized = CapitalizeDecorator.new(@student)
+      expect(@capitalized.correct_name).to eql 'Alexis'
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the test for the `CapitalizeDecorator` class passes as it currently throws an error that does not identify the classroom constant.
![Screenshot from 2022-07-28 00-21-58](https://user-images.githubusercontent.com/58692590/181389124-0f921a13-07a6-4f65-aee6-dbb936c16d7d.png)